worldedit-core/src/main/java/com/sk89q/worldedit/session/SessionManager.java

```diff
@@ -155,9 +155,7 @@ public synchronized LocalSession get(SessionOwner owner) {
             session.setBlockChangeLimit(config.defaultChangeLimit);
 
             // Remember the session if the session is still active
```

Comment:
You should probably change this comment, it no longer explains the code correctly.